content/hi/docs/concepts/architecture/cgroups.md

```diff
@@ -0,0 +1,125 @@
+---
+title: सीग्रुप v2 के बारे में
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+Linux पर, {{< glossary_tooltip text="control groups" term_id="cgroup" >}}
+प्रक्रियाओं को आवंटित संसाधनों को सीमित करें।
+
+{{< glosary_tooltip text="kubelet" term_id="kubelet" >}} और
```

Comment:
There is a typo in the `glossary_tooltip` shortcode name _(leading to [build failures](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/65723bda5503fb00084bbf4a))_. Additionally, please consider localizing the "text" parameter passed to the shortcode to Hindi instead of using English. Refer the below suggestion.

```suggestion
{{< glossary_tooltip text="क्यूबलेट" term_id="kubelet" >}} और
```